index.js

```diff
@@ -83,15 +85,25 @@ const MultiselectPrompt = {
 		this.render()
 	}
 	, right: function () {
-		this.value[this.cursor].selected = true
-		this.render()
+		if (this.value.filter (e => e.selected).length < this.maxChoices) {
```

Comment:
Sorry @derhuerst I'm not quite sure I'm following how can you make 

```Javascript
if (this.value.filter (e => e.selected).length < this.maxChoices) {
			this.value[this.cursor].selected = true
			this.render()
		}
		else
			return this.bell()
```
fail faster? It checks if the number of selected items is inferior than `maxChoices`, if not it fails `return this.bell()` immediately.
I'm not sure what you'd like :/